scico/blockarray.py

```diff
@@ -289,17 +289,24 @@
 Between BlockArray and :class:`.LinearOperator`
 ***********************************************
 
-.. todo::
-   Improve this
 
 The :class:`.Operator` and :class:`.LinearOperator` classes are designed
-to work on :class:`.BlockArray`. The shapes must conform:
+to work on :class:`.BlockArray`s. For example
```

Comment:
Perhaps amend slightly to make it clear that they don't *only* work on `BlockArrray`s?